files/zh-cn/web/api/credential/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Credential
+slug: Web/API/Credential
+---
```

Comment:
Let's add the sourceCommit [metadata](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E5%85%83%E6%95%B0%E6%8D%AE)

```suggestion
l10n:
  sourceCommit: 952db471d65c30302c44388edf3c0aa094874319
---
```